docs/devnet.md

```diff
@@ -1,37 +1,40 @@
 ---
-description: The IOTA foundation provides load-balanced public devnet endpoints, with MQTT and the HTTP REST API enabled.
+description: The IOTA foundation provides load-balanced public devnet endpoints, where MQTT and the HTTP REST API enabled.
```

Comment:
Something is missing here. API is/are enabled?

Reply:
"Are" 👍